core/modules/czar/CzarConfig.cc

```diff
@@ -38,6 +41,22 @@ namespace {
 
 LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.CzarConfig");
 
+void QservLogger(struct timeval const& mtime,
```

Comment:
This function is almost identical to one XrdLogging.cc. As it does not have direct dependency on xrootd we can probably move it on a common place, for example `util` module (maybe find it a better name).